package.json

```diff
@@ -24,6 +24,7 @@
     "build-lr": "yarn reset-link && node ./build/build-lightrider-bundles.js",
     "build-pack": "bash build/build-pack.sh",
     "build-report": "node build/build-report.js",
+    "build-dist-reports": "yarn build-report && node lighthouse-core/scripts/build-report-for-autodeployment.js",
```

Comment:
bike shed: what is `build-dist-reports` referring to? Just build reports that will be in `dist/`? Because it's also sample reports in particular. `build-sample-reports`?

Reply:
hehe me and paul were talking about the naming here...

* `build-dist-reports`: Builds the report renderer and the reports for the Vercel deployment
* `build-report-for-autodeployment.js`: Builds the reports for the Vercel deployment
* `now-build`: The Vercel deployment used to be called "now". not anymore

some renaming is in order all around.